File: CS/WindowsFormsApplication1/WPF/ReportPreviewModel.cs
using DevExpress.Xpf.Printing;

namespace WindowsFormsApplication1.WPF {
    public class ReportPreviewModel : XtraReportPreviewModel {
        protected override bool CanPageSetup(object parameter) {
            return false;
        }
        protected override bool CanScale(object parameter) {
            return false;
        }
    }
}

File: VB/WindowsFormsApplication1/WPF/ReportPreviewModel.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpf.Printing

Namespace WindowsFormsApplication1.WPF
	Public Class ReportPreviewModel
		Inherits XtraReportPreviewModel
		Protected Overrides Function CanPageSetup(ByVal parameter As Object) As Boolean
			Return False
		End Function
		Protected Overrides Function CanScale(ByVal parameter As Object) As Boolean
			Return False
		End Function
	End Class
End Namespace
